api/converters_test.py

```diff
@@ -606,10 +606,16 @@ def test_maxmimal(self, mock_now):
           'is_api_polyfill': False,
           'is_language_polyfill': True,
           'is_same_origin_css': False,
+          'covers_existance': False,
```

Comment:
small error, but will need to be changed in a few places:
```suggestion
          'covers_existence': False,
```